app/screens/channel/channel_nav_bar/channel_drawer_button/__snapshots__/channel_drawer_button.test.js.snap

```diff
@@ -1,12 +1,11 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ChannelDrawerButton should match, full snapshot 1`] = `
-<TouchableOpacity
+<Component
   accessibilityHint="Opens the channels and teams drawer"
   accessibilityLabel="Channels and teams"
   accessibilityRole="button"
   accessible={true}
-  activeOpacity={0.2}
```

Comment:
Possible styling loss?

(leaving this as a catch-all review comment/question for all instances of possible styling loss on components in this PR)

Reply:
Touchable